sources/packages/backend/apps/api/src/auth/roles.enum.ts

```diff
@@ -31,6 +31,7 @@ export enum Role {
   InstitutionApproveDeclineDesignation = "institution-approve-decline-designation",
   InstitutionApproveDeclineOfferingChanges = "institution-approve-decline-offering-changes",
   InstitutionApproveDeclineApplicationOfferingChangeRequest = "institution-approve-decline-application-offering-change-request",
+  InstitutionDeactivateEducationProgram = "institution-deactivate-education-program",
```

Comment:
Minor. Should we have the role named in generic context. something like `institution-update-program-active-status` ?

Reply:
We can, but we are not sure how we will handle the activation if we ever do. For now, it reflects what the feature does.
I was trying to keep the controller's layer targeting the deactivation feature.